packages/components/src/components/SearchInput.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { Input, type InputProps } from '@ttoss/ui';
+import { useDebounce } from '@ttoss/react-hooks';
+
+export type SearchInputProps = Omit<InputProps, 'onChange'> & {
+  loading?: boolean;
+  debounce?: number;
+  onChange: (value?: string | number | readonly string[]) => void;
```

Comment:
What is the case for `readonly string[]`?

Reply:
It was to follow the value's type, I updated to use it from `InputProps` type